Lib/test/test_deque.py

```diff
@@ -288,6 +288,11 @@ def test_index(self):
                     else:
                         self.assertEqual(d.index(element, start, stop), target)
 
+        # Test large start argument
+        d = deque(range(0, 10000, 10))
```

Comment:
Is `leftindex == 0` in this case? Would be nice to test also cases when `leftindex != 0` and when `(leftindex + start) % BLOCKLEN < leftindex`.